Products/CMFPlone/browser/templates/author.pt

```diff
@@ -219,7 +219,7 @@
 
                           <p>
                           <a href=""
-                             tal:attributes="href string:$here_url/search?Creator=${username}&amp;sort_on=created&amp;sort_order=reverse"
+                             tal:attributes="href string:$here_url/search?Creator=${view.makeQuery(username)}&amp;sort_on=created&amp;sort_order=reverse"
                              i18n:translate="go_to_search_author_content">
```

Comment:
This gives an error when I view the author page.  Also, we were using an older way of spelling this, using `tal:attributes`, which can be made shorted and more readable.  Can you change it to the following?  Then it works for me.

```
                          <a href="${here_url}/search?${python:view.makeQuery(Creator=username, sort_on='created', sort_order='reverse')}"
                             i18n:translate="go_to_search_author_content">
```

Reply:
@mauritsvanrees now please verify